src/pages/en/core-concepts/astro-components.md

```diff
@@ -154,25 +154,26 @@ const items = ["Dog", "Cat", "Platypus"];
 
 #### Fragments & Multiple Elements
 
-Remember: an Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`.
-
-However, when using an Astro JSX-like expression to dynamically create elements, you must wrap these multiple elements inside of a **Fragment** just like you would in JavaScript or JSX. Astro supports using either `<Fragment> </Fragment>` or `<> </>`.
+An Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`, unlike JavaScript or JSX.
```

Comment:
May be overkill but what do you think of an extra example here?

````suggestion
An Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`, unlike JavaScript or JSX.

```astro
---
// Template with multiple elements
---
<p>No need to wrap elements in a single containing element.</p>
<p>Astro supports multiple root elements in a template.</p>
```

However, when using an expression to dynamically create multiple elements, you should wrap these elements inside a **Fragment** as you would in JavaScript or JSX. Astro supports using either `<Fragment> </Fragment>` or the shorthand `<> </>`.
````

Reply:
(Probably going to amend with a lower-case fragment in the text, as I don't think it would normally be capitalized when not the component itself?)